backend/battle/tests.py

```diff
@@ -1,3 +1,29 @@
-# from django.test import TestCase
 
-# Create your tests here.
+from django.test import TestCase, Client
```

Comment:
Please create a folder for tests and name this according to the module being tested

Reply:
Got it :)